src/app/api/clients/[id]/route.ts

```
@@ -0,0 +1,118 @@
+            );
+        }
+
+        return NextResponse.json(client);
+    });
+}
+
+export async function PUT(
+    request: NextRequest,
+    { params }: { params: Params }
+) {
+    return withRouteMiddleware(request, async ({ }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)